app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -186,27 +195,30 @@ private fun ConversationScreen(
             content = {
                 Scaffold(
                     topBar = {
-                        ConversationScreenTopAppBar(
-                            title = conversationName.ifEmpty { stringResource(id = R.string.member_name_deleted_label) },
-                            avatar = {
-                                when (conversationAvatar) {
-                                    is ConversationAvatar.Group ->
-                                        GroupConversationAvatar(
-                                            color = colorsScheme().conversationColor(id = conversationAvatar.conversationId)
+                        Column {
+                            CommonTopAppBar(commonTopAppBarViewModel = commonTopAppBarViewModel as CommonTopAppBarViewModel)
```

Comment:
This cast is needed because in `ConversationScreen` we receive its abstract class and not the "implementation" itself just in order so we can use `@Preview` from Compose, otherwise it would not be needed.